pkg/docs/data/description.tpl

```diff
@@ -5,13 +5,34 @@
 ## Documentation
 
 This image is maintained by [KubeVirt](https://kubevirt.io/) and automatically created from https://github.com/kubevirt/containerdisks.
-
-For how to get started with `KubeVirt` visit the [UserGuide](https://kubevirt.io/user-guide/)
+<br />
+<br />
+For how to get started with `KubeVirt` visit the [user guide](https://kubevirt.io/user-guide/):
   * [Installation](https://kubevirt.io/user-guide/operations/installation/#installing-kubevirt-on-kubernetes)
-  * [ContainerDisks](https://kubevirt.io/user-guide/virtual_machines/disks_and_volumes/#containerdisk)
+  * [Containerdisks](https://kubevirt.io/user-guide/virtual_machines/disks_and_volumes/#containerdisk)
+  * [virtctl](https://kubevirt.io/user-guide/user_workloads/virtctl_client_tool)
+  * [Creating VirtualMachines by using virtctl](https://kubevirt.io/user-guide/user_workloads/creating_vms/#creating-virtualmachines-by-using-virtctl)
+
+## Examples
+
+### Creating a VirtualMachine and importing this containerdisk with virtctl
+
+You can create a VirtualMachine that will import this containerdisk by running the following command:
+
+```shell
+virtctl create vm --volume-import=type:registry,url:docker://{{ .Image }},size:10Gi | kubectl create -f -
```

Comment:
This doesn't infer the defaults from the containerdisk or datavolume FWIW so we should still include the instance type and preference here for now.

@0xFelix Did we ever track this as a bug anywhere? Would be good justification to move infer into the controller sync loop this cycle as I think that might be the only way we can fix this post DV creation.

Reply:
It does infer the defaults from the containerdisk because `virtctl` tellss the API to do so at the moment.

Reply:
Okay, now that I'm thinking of it again. Not it can't be inferred because the DV is not imported yet on creation of the VM. You are right. Let me fix it.